docs/process/releasing-updates.md

```diff
@@ -71,18 +71,20 @@ Run the script below (which relies on the your personal access token being set),
 For `production` and `beta` releases, run:
 
 ```shellsession
-$ yarn draft-release (production|beta)
+$ yarn draft-release (production|beta|test)
 ```
 
 If you are creating a new beta release, the `yarn draft-release beta` command will help you find the new release entries for the changelog.
 
 If you are create a new `production` release, you should just combine and sort the previous `beta` changelog entries.
 
-(For `test` releases, follow the directions in the steps below to update `app/package.json`'s `version` to a higher version and add a changelog entry. The script does not support test yet.)
+(For `test` releases, it will do a naive attempt to update the `app/package.json`'s `version` and add a changelog entry. You may have to adjust the test version and add a changelog entry to the `changelog.json`.)
```

Comment:
This is not true anymore, right? With your latest changes it should always work (or at least as much as it does for beta and production builds).